ntbs-service/Pages/Migration/Index.cshtml.cs

```diff
@@ -33,10 +48,49 @@ public IActionResult OnGetAsync()
 
         public async Task<IActionResult> OnPostAsync()
         {
+            if (!ModelState.IsValid)
+            {
+                return Page();
+            }
+
             var requestId = HttpContext.TraceIdentifier;
 
-            Results = await _service.ImportNotificationsAsync(requestId, NotificationId);
+            if (UploadedFile != null)
+            {
+                var notificationIds = await GetIdListFromFile(UploadedFile);
+                var IdGroups = splitList(notificationIds, 10);

```

Comment:
Is the purpose of this line to only get the first 10 records in the file? If so I think the `splitList` method is a little overkill. If not and it's just for the purpose of batching them into 10s, and it's also meant to be iterated over - is there a motivation for not just iterating over notificationIds?

Reply:
it should be 1000 actually, I need to remove this from here

Reply:
If we only want the first batch of 1000 notificationIds, then I think splitList is still a bit overkill?